janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -163,9 +147,16 @@
             new ConfigOption<>(ES_CREATE_NS, "use-external-mappings",
             "Whether JanusGraph should make use of an external mapping when registering an index.", ConfigOption.Type.MASKABLE, false);
 
+    public static final ConfigOption<Boolean> USE_ALL_FIELD =
+        new ConfigOption<>(ELASTICSEARCH_NS, "use-all-field",
+            "Whether JanusGraph should add an \"all\" field mapping. When enabled field mappings will " +
+            "include a \"copy_to\" parameter referencing the \"all\" field. This is supported since Elasticsearch 6.x " +
+            " and is required when using wildcard fields starting in Elasticsearch 6.x.", ConfigOption.Type.GLOBAL_OFFLINE, true);
+
```

Comment:
Copy_to has been added in ES 2, so your code should work in ES 2, 5 and 6 ?
And where do you want to use wildcard fields ? In query_string ?

Reply:
@davidclement90 It comes up in [JanusGraphIndexTest#testWidcardQuery](https://github.com/JanusGraph/janusgraph/blob/1563460babc6ebd658ac3dd4c687d7abdebb82b2/janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphIndexTest.java#L1667-L1689).